### ISSUE TITLE
PADSTOW: Create a PDB from the SEED PDB.sql fails with ORA-00942

### ISSUE THREAD

**Issue by dfhawthorne** [dfhawthorne/sql-ocm12c#2]
After running [`Create a PDB from the SEED PDB.sql`](https://github.com/dfhawthorne/sql-ocm12c/blob/master/General%20Database%20and%20Network%20Administration%2C%20and%20Backup%20Strategy/Create%20and%20manage%20pluggable%20databases/Create%20a%20PDB%20from%20the%20SEED%20PDB.sql), I get the following messages:
```
ERROR:
ORA-00942: table or view does not exist


Error accessing PRODUCT_USER_PROFILE
Warning:  Product user profile information not loaded!
You may need to run PUPBLD.SQL as SYSTEM
```

**Comment by dfhawthorne** [comment]
Fixed via:
```
connect system
alter session set container=plum;
@/opt/app/oracle/product/12.1.0/dbhome_1/sqlplus/admin/pupbld.sql
```

**Comment by dfhawthorne** [comment]
Put fix into script via commit cda1192
